l_upper_trunk: 0.510
l_upper_arm: 0.500
l_forearm: 0.299
l_hand: 0.203
